diff --git a/node_modules/react-scripts/scripts/build.js b/node_modules/react-scripts/scripts/build.js
index 7a3a635..002d39c 100644
--- a/node_modules/react-scripts/scripts/build.js
+++ b/node_modules/react-scripts/scripts/build.js
@@ -20,15 +20,15 @@ process.on('unhandledRejection', err => {
 });
 
 // Ensure environment variables are read.
-require('../config/env');
+require('react-scripts/config/env');
 
 const path = require('path');
 const chalk = require('react-dev-utils/chalk');
 const fs = require('fs-extra');
 const bfj = require('bfj');
 const webpack = require('webpack');
-const configFactory = require('../config/webpack.config');
-const paths = require('../config/paths');
+const configFactory = require('react-scripts/config/webpack.config');
+const paths = require('react-scripts/config/paths');
 const checkRequiredFiles = require('react-dev-utils/checkRequiredFiles');
 const formatWebpackMessages = require('react-dev-utils/formatWebpackMessages');
 const printHostingInstructions = require('react-dev-utils/printHostingInstructions');
@@ -36,7 +36,7 @@ const FileSizeReporter = require('react-dev-utils/FileSizeReporter');
 const printBuildError = require('react-dev-utils/printBuildError');
 
 const measureFileSizesBeforeBuild =
-  FileSizeReporter.measureFileSizesBeforeBuild;
+    FileSizeReporter.measureFileSizesBeforeBuild;
 const printFileSizesAfterBuild = FileSizeReporter.printFileSizesAfterBuild;
 const useYarn = fs.existsSync(paths.yarnLockFile);
 
@@ -55,89 +55,97 @@ const argv = process.argv.slice(2);
 const writeStatsJson = argv.indexOf('--stats') !== -1;
 
 // Generate configuration
-const config = configFactory('production');
+const initConfig = configFactory('production');
+const overrides = require("./config-overrides");
+const config = overrides(initConfig);
 
 // We require that you explicitly set browsers and do not fall back to
 // browserslist defaults.
 const { checkBrowsers } = require('react-dev-utils/browsersHelper');
 checkBrowsers(paths.appPath, isInteractive)
-  .then(() => {
-    // First, read the current file sizes in build directory.
-    // This lets us display how much they changed later.
-    return measureFileSizesBeforeBuild(paths.appBuild);
-  })
-  .then(previousFileSizes => {
-    // Remove all content but keep the directory so that
-    // if you're in it, you don't end up in Trash
-    fs.emptyDirSync(paths.appBuild);
-    // Merge with the public folder
-    copyPublicFolder();
-    // Start the webpack build
-    return build(previousFileSizes);
-  })
-  .then(
-    ({ stats, previousFileSizes, warnings }) => {
-      if (warnings.length) {
-        console.log(chalk.yellow('Compiled with warnings.\n'));
-        console.log(warnings.join('\n\n'));
-        console.log(
-          '\nSearch for the ' +
-            chalk.underline(chalk.yellow('keywords')) +
-            ' to learn more about each warning.'
-        );
-        console.log(
-          'To ignore, add ' +
-            chalk.cyan('// eslint-disable-next-line') +
-            ' to the line before.\n'
-        );
-      } else {
-        console.log(chalk.green('Compiled successfully.\n'));
-      }
-
-      console.log('File sizes after gzip:\n');
-      printFileSizesAfterBuild(
-        stats,
-        previousFileSizes,
-        paths.appBuild,
-        WARN_AFTER_BUNDLE_GZIP_SIZE,
-        WARN_AFTER_CHUNK_GZIP_SIZE
-      );
-      console.log();
-
-      const appPackage = require(paths.appPackageJson);
-      const publicUrl = paths.publicUrlOrPath;
-      const publicPath = config.output.publicPath;
-      const buildFolder = path.relative(process.cwd(), paths.appBuild);
-      printHostingInstructions(
-        appPackage,
-        publicUrl,
-        publicPath,
-        buildFolder,
-        useYarn
-      );
-    },
-    err => {
-      const tscCompileOnError = process.env.TSC_COMPILE_ON_ERROR === 'true';
-      if (tscCompileOnError) {
-        console.log(
-          chalk.yellow(
-            'Compiled with the following type errors (you may want to check these before deploying your app):\n'
-          )
-        );
-        printBuildError(err);
-      } else {
-        console.log(chalk.red('Failed to compile.\n'));
-        printBuildError(err);
-        process.exit(1);
+    .then(() => {
+      // First, read the current file sizes in build directory.
+      // This lets us display how much they changed later.
+      return measureFileSizesBeforeBuild(paths.appBuild);
+    })
+    .then(previousFileSizes => {
+      // Remove all content but keep the directory so that
+      // if you're in it, you don't end up in Trash
+      fs.emptyDirSync(paths.appBuild);
+      // Merge with the public folder
+      copyPublicFolder();
+      // Start the webpack build
+      return build(previousFileSizes);
+    })
+    .then(
+        ({ stats, previousFileSizes, warnings, compiler }) => {
+          if (warnings.length) {
+            console.log(chalk.yellow('Compiled with warnings.\n'));
+            console.log(warnings.join('\n\n'));
+            console.log(
+                '\nSearch for the ' +
+                chalk.underline(chalk.yellow('keywords')) +
+                ' to learn more about each warning.'
+            );
+            console.log(
+                'To ignore, add ' +
+                chalk.cyan('// eslint-disable-next-line') +
+                ' to the line before.\n'
+            );
+          } else {
+            console.log(chalk.green('Compiled successfully.\n'));
+          }
+
+          console.log('File sizes after gzip:\n');
+          printFileSizesAfterBuild(
+              stats,
+              previousFileSizes,
+              paths.appBuild,
+              WARN_AFTER_BUNDLE_GZIP_SIZE,
+              WARN_AFTER_CHUNK_GZIP_SIZE
+          );
+          console.log();
+
+          const appPackage = require(paths.appPackageJson);
+          const publicUrl = paths.publicUrlOrPath;
+          const publicPath = config.output.publicPath;
+          const buildFolder = path.relative(process.cwd(), paths.appBuild);
+          printHostingInstructions(
+              appPackage,
+              publicUrl,
+              publicPath,
+              buildFolder,
+              useYarn
+          );
+          compiler.close(err => {
+            if (err && err.message) {
+              console.log(err.message);
+              process.exit(1);
+            }
+          });
+        },
+        err => {
+          const tscCompileOnError = process.env.TSC_COMPILE_ON_ERROR === 'true';
+          if (tscCompileOnError) {
+            console.log(
+                chalk.yellow(
+                    'Compiled with the following type errors (you may want to check these before deploying your app):\n'
+                )
+            );
+            printBuildError(err);
+          } else {
+            console.log(chalk.red('Failed to compile.\n'));
+            printBuildError(err);
+            process.exit(1);
+          }
+        }
+    )
+    .catch(err => {
+      if (err && err.message) {
+        console.log(err.message);
       }
-    }
-  )
-  .catch(err => {
-    if (err && err.message) {
-      console.log(err.message);
-    }
-    process.exit(1);
-  });
+      process.exit(1);
+    });
 
 // Create the production build and print the deployment instructions.
 function build(previousFileSizes) {
@@ -157,8 +165,8 @@ function build(previousFileSizes) {
         // Add additional information for postcss errors
         if (Object.prototype.hasOwnProperty.call(err, 'postcssNode')) {
           errMessage +=
-            '\nCompileError: Begins at CSS selector ' +
-            err['postcssNode'].selector;
+              '\nCompileError: Begins at CSS selector ' +
+              err['postcssNode'].selector;
         }
 
         messages = formatWebpackMessages({
@@ -167,7 +175,7 @@ function build(previousFileSizes) {
         });
       } else {
         messages = formatWebpackMessages(
-          stats.toJson({ all: false, warnings: true, errors: true })
+            stats.toJson({ all: false, warnings: true, errors: true })
         );
       }
       if (messages.errors.length) {
@@ -179,21 +187,21 @@ function build(previousFileSizes) {
         return reject(new Error(messages.errors.join('\n\n')));
       }
       if (
-        process.env.CI &&
-        (typeof process.env.CI !== 'string' ||
-          process.env.CI.toLowerCase() !== 'false') &&
-        messages.warnings.length
+          process.env.CI &&
+          (typeof process.env.CI !== 'string' ||
+              process.env.CI.toLowerCase() !== 'false') &&
+          messages.warnings.length
       ) {
         // Ignore sourcemap warnings in CI builds. See #8227 for more info.
         const filteredWarnings = messages.warnings.filter(
-          w => !/Failed to parse source map/.test(w)
+            w => !/Failed to parse source map/.test(w)
         );
         if (filteredWarnings.length) {
           console.log(
-            chalk.yellow(
-              '\nTreating warnings as errors because process.env.CI = true.\n' +
-                'Most CI servers set it automatically.\n'
-            )
+              chalk.yellow(
+                  '\nTreating warnings as errors because process.env.CI = true.\n' +
+                  'Most CI servers set it automatically.\n'
+              )
           );
           return reject(new Error(filteredWarnings.join('\n\n')));
         }
@@ -203,13 +211,14 @@ function build(previousFileSizes) {
         stats,
         previousFileSizes,
         warnings: messages.warnings,
+        compiler
       };
 
       if (writeStatsJson) {
         return bfj
-          .write(paths.appBuild + '/bundle-stats.json', stats.toJson())
-          .then(() => resolve(resolveArgs))
-          .catch(error => reject(new Error(error)));
+            .write(paths.appBuild + '/bundle-stats.json', stats.toJson())
+            .then(() => resolve(resolveArgs))
+            .catch(error => reject(new Error(error)));
       }
 
       return resolve(resolveArgs);
